src/EA.Iws.Web/Areas/NotificationMovements/Controllers/PrenotificationBulkUploadController.cs

```diff
@@ -109,27 +109,26 @@ public async Task<ActionResult> UploadPrenotifications(Guid notificationId, Pren
                 return View(model);
             }
 
-            var validationSummary = await validator.GetValidationSummary(model.File, notificationId);
-
+            var validationSummary = await validator.GetPrenotificationValidationSummary(model.File, notificationId);
             var failedFileRules = validationSummary.FileRulesResults.Where(r => r.MessageLevel == MessageLevel.Error).Select(r => r.Rule).ToList();
             var failedContentRules = validationSummary.ContentRulesResults.Where(r => r.MessageLevel == MessageLevel.Error).ToList();
             var warningContentRule = validationSummary.ContentRulesResults.Where(r => r.MessageLevel == MessageLevel.Warning).ToList();
-
             model.FailedFileRules = failedFileRules;
             model.FailedContentRules = failedContentRules;
             model.WarningContentRules = warningContentRule;
+            var shipments = validationSummary.ShipmentNumbers != null ? validationSummary.ShipmentNumbers.ToList() : null;
 
-            TempData["PrenotificationShipments"] = validationSummary.ShipmentNumbers;
-            TempData["PreNotificationFileName"] = model.File.FileName;
+            var shipmentsModel = new ShipmentMovementDocumentsViewModel(notificationId, shipments, model.File.FileName);
 
             if (model.ErrorsCount > 0 || model.WarningsCount > 0)
             {
+                TempData["PrenotificationShipments"] = shipments;
+                TempData["PreNotificationFileName"] = model.File.FileName;
                 return View("Errors", model);
             }
 
-            var shipmentsModel = new ShipmentMovementDocumentsViewModel(notificationId, validationSummary.ShipmentNumbers, model.File.FileName);
-
-            TempData["DraftBulkUploadId"] = validationSummary.DraftBulkUploadId;
```

Comment:
The DraftBulkUploadId key is not being set anymore - this is needed on the POST Documents action when calling the `Mediator` with the request `CreateBulkPrenotification`.

Reply:
Noticed, and added back into the post